### PR TITLE
Added slider component

### PR DESCRIPTION
**Pull request by rutu14**
Contains two variations: with label and without label.

### CODE REVIEW

File: documentation/css/style.css [hunk 1]
```diff
@@ -210,6 +210,10 @@
     height: 20rem;
 }
 
+.slider-height{
+    height: 9rem;
+}
+
 .container-input{
     display: flex;
```
Comment (sach10-create) [inline documentation/css/style.css:218]: can be clubbed into a class, as it is used at various places

**Review by sach10-create — APPROVED** [review]
pixels can be changed to rems, wherever possible, rest code looks clean.
Great Work!

File: documentation/css/style.css (hunk 1 above)
Comment (rutu14) [reply documentation/css/style.css:218]: Sure

**Review by srishtigautam21 — APPROVED** [review]
good to merge

**Review by varun13169 — APPROVED** [review]
Appreciation: Code looks clean and good, so does the UI

**Pull request merged**